Controller/SuggestionProviderController.php

```diff
@@ -47,8 +78,9 @@ public function showAction(
         $_format = 'html'
     ) {
         $code = $exception->getStatusCode();
+        $templateForSuggestion = $this->getTemplateForSuggestions($_format);
```

Comment:
i would move this to after the "404 || request matcher" check and have a separate return. we should not do the lookup if we do not handle this anyways.